LEAF_Nexus/scripts/refreshOrgchartEmployees.php

```diff
@@ -243,20 +254,33 @@ function updateEmployeeDataBatch(array $localEmployeeUsernames = [])
     if (empty($orgEmployeeDataRes)) {
         return FALSE;
     }
+
+    // regather employee data since we may have inserted items
+    list($localEmpUIDs,$localEmpArray) = getEmployeeUIDs($localEmployeeUsernames);
+
     foreach ($orgEmployeeDataRes as $orgEmployeeData) {
 
-        $localEmployeeDataArray[] = [
-            // this will need to be checked into further I cannot silence this warning without possibly breaking things.
-            'empUID' => $localEmpArray[$orgEmployeeData['userName']],
-            'indicatorID' => $orgEmployeeData['indicatorID'],
-            'data' => $orgEmployeeData['data'],
-            'author' => $orgEmployeeData['author'],
-            'timestamp' => $orgEmployeeData['timestamp'],
-        ];
-    }
+        // if this user is not found, we will skip adding data for them.
+        if(empty($localEmpArray[$orgEmployeeData['userName']]) == true ){
```

Comment:
Do we need the == true in this case? the empty method will return true or false.

Reply:
I know I harp on comparing things correctly.  In this case Jamie is correct.  empty() returns true or false so is perfectly fine just being directly referenced.

Reply:
removed true, had it since it made it very clear we are looking at true/false.